.travis.yml

```diff
@@ -4,18 +4,18 @@ dist: bionic
 sudo: false
 
 python:
-  - '3.5'
+  # TODO: Make sure 3.6 is what is running on the servers
```

Comment:
^ needs confirmation

Reply:
@raylu confirmed 3.7

Reply:
Really?

Reply:
as mentioned amidst all my discord spam, we're 3.7 everywhere on DO, even for osr-bot. py3.5 has been uninstalled